annotation-server/test/.env.example

```diff
@@ -1,2 +1,4 @@
-DRUGBANK_ZIP=data/random-database.zip
-DRUGBANK_XML=random-database.xml
+DRUGBANK_ZIP=data/example-database.zip
```

Comment:
(Full-on review coming soon, just for now:) I just found out that [using multiple ``dotenv``s is discouraged](https://www.npmjs.com/package/dotenv#faq). Is there another way to do this?

Reply:
I think it's fair to use different `.env`s in different environments ([NestJS ref](https://docs.nestjs.com/techniques/configuration)), i.e. development/production vs testing in our case. In production and development only the root `.env` is used; we could use only `test/.env` in testing, but this would mean we would have to redefine the variables we want to reuse from the dev/prod environment creating a second source of truth and something people can forget to set up / keep up to date. That's why we decided to use `test/.env` on top of `.env`.

Reply:
I think the `test/.env` ist only needed for local development/testing, as the GitHub workflow which tests the code is defining its environment variables without any `.env`-file. So if you can think of another way to fit our needs, please let us know, but as @jannis-baum said, this seems like the best way to do local testing without resetting the `.env` configuration every time you switch between testing and development.

Reply:
On another note we could of course introduce new variables like `DRUGBANK_ZIP_TEST` into the main `.env` and use those for testing; this would add some code overhead (checking `proc.env.NODE_ENV === 'test'` where we need to decide which variable to use) but in turn void the need for a second `.env` and (maybe?) make the single `.env` more *übersichtlich*. I don't personally have a strong opinion on this, maybe someone else does.

Reply:
As you pointed out, I was mistaken. It seems to be very fair to use different ``.env`` files in different environments, as this is their purpose. But according to [the 12 factor app](https://12factor.net/config), the variables in the different ``.env``s should actually be different. Otherwise, a global config should be chosen.  
l think it is fair to argue that we object to this and prefer a single source of truth, using the variables is the ``.env`` per default and overwriting them with vars in ``test/.env`` in tests, leaving everything as it is.
Also, this seems to be of lower priority.